tests/previousresults/getrequest_public-:collectionId-:displayId_bbapublic.html

```diff
@@ -178,7 +178,7 @@ <h5>
      <h4>
       LuxR protein generator
       <br/>
-      <a href="/search/?q=LuxR protein generator&lt;br/&gt;
+      <a href="/search/?q=LuxR protein generator 
```

Comment:
How did this change?

Reply:
There were extra br characters getting added to the description/